pyproject.toml

```diff
@@ -26,6 +26,8 @@ qiskit-device-benchmarking = { path = "submodules/qiskit-device-benchmarking" }
 qiskit-ibm-runtime = "^0.34.0"
 scipy = "^1.15.1"
 tabulate = "^0.9.0"
+qiskit-qasm3-import = "^0.5.1"
```

Comment:
Do we need this? I just removed it, because `deptry` was telling me that it's not used.

Reply:
Probably came about through some experimentation/debugging. Removed in 00032dddd85973f717e01513c816bd562f3f68b4